kobo/settings/base.py

```diff
@@ -1091,14 +1093,15 @@ def dj_stripe_request_callback_method():
     if region := env.str('AWS_S3_REGION_NAME', False):
         AWS_S3_REGION_NAME = region
 
-
 ''' Storage configuration '''
+default_file_storage = 'django.core.files.storage.FileSystemStorage'
```

Comment:
You could refer to the django default via `django.conf.global_settings` instead of redefining the setting.